### PR TITLE
feature: remove RecIt personalization

### PR DESCRIPTION
**Pull request by mmiermans**
# Goal
Remove dependency on RecIt.

## Review
@lfishhead Could you review the changes in the JSON files?

## Reference

https://mozilla-hub.atlassian.net/browse/MC-20

## Implementation details
- I cautiously removed some slates and lineups, which I am confident are not actively used.

### CODE REVIEW

File: app/json/slate_configs.json [hunk 1]
```diff
@@ -260,43 +260,6 @@
       }
     ]
   },
-  {
-    "id": "631d8077-1462-4397-ad0a-aa340c27570a",
```
Comment (mmiermans) [inline app/json/slate_configs.json:264]: [Github search shows no client usage](https://github.com/search?q=org%3APocket%20aa340c27570a&type=code)

File: app/json/slate_configs.json [hunk 2]
```diff
@@ -260,43 +260,6 @@
...
-        "rankers": [
-          "top30-items",
-          "thompson-sampling-28day",
-          "pubspread"
-        ]
-      }
-    ]
-  },
-  {
-    "id": "ad73c061-5ac5-458f-8691-ab071f7eadd7",
```
Comment (mmiermans) [inline app/json/slate_configs.json:283]: [Github search shows no client usage](https://github.com/search?q=org%3APocket%20ab071f7eadd7&type=code)

File: app/json/slate_lineup_configs.json [hunk 3]
```diff
@@ -550,69 +514,15 @@
   },
   {
     "id": "05027beb-0053-4020-8bdc-4da2fcc0cb68",
```
Comment (mmiermans) [inline app/json/slate_lineup_configs.json:516]: I see a [reference to this id in web-qa](https://github.com/Pocket/web-qa/blob/242705ab9e69c2939ecc4fff18f32405a7fa46e2/scripts/generators/seed.js#L33), so I'm inclined to keep this lineup around for now.

File: app/json/slate_lineup_configs.json [hunk 4]
```diff
@@ -35,29 +33,13 @@
...
-          "44dff273-3604-40e9-a0b5-bf3852581990",
-          "494eaa68-f048-4f52-bb9a-5e7d2526f5cb",
-          "e6600ffe-b65a-42d3-b570-f460f3d7326e",
-          "72ce0827-9dbc-470f-9722-39476daaeb0f",
-          "9901851d-ea76-4667-8dc7-0c1677ecb910",
-          "ab6048ca-5c61-4e87-aefd-7a612b60ab7b",
-          "8a495385-9972-4d0e-b93e-4234f4897939",
-          "fa61096a-b681-4251-b299-2fda06c49ebf",
-          "47688fcf-2bbb-4e0e-a02e-1f68c248a67e",
-          "d024ce9c-ed96-453f-a81e-8a0b850681e7",
```
Comment (mmiermans) [inline app/json/slate_lineup_configs.json:60]: I believe these are all topic slates. I arbitrarily kept the 'business' one at the top. Should only be use by a small fraction of users who are still on an old client.

File: app/json/slate_lineup_configs.json [hunk 5]
```diff
@@ -68,29 +50,13 @@
...
-          "951ae6c3-4438-458b-9936-7f3b8ff0f178",
-          "47023d21-0aa6-4f98-9077-eac21fad44f1",
-          "1a8de2be-db03-4a2f-901a-7c4c1cbd156a",
-          "f4b58337-4ab6-4563-9503-c384e773946f",
-          "f8c945b0-52fa-4203-bb31-6747245fe021",
-          "b548b456-70c4-474d-a723-80d040d00fec",
-          "5674f085-07b6-43c3-bcde-2774db3f6384",
-          "e6c00454-4a00-47c1-8f22-906122c261ed",
-          "ed9604ce-b752-48bd-b8c5-3a8cf6b54ced",
-          "4345efa7-2c89-4884-b836-3260757a3a97",
```
Comment (mmiermans) [inline app/json/slate_lineup_configs.json:93]: I believe these are all topic slates. I arbitrarily kept the 'business' one at the top. Should only be use by a small fraction of users who are still on an old client.

File: app/json/slate_lineup_configs.json [hunk 6]
```diff
@@ -550,69 +514,15 @@
...
-          "6d1273a5-055e-4de0-8a5b-5f2b79d37e5c",
-          "ea40bef5-4406-488d-ad9d-915dfa1f0794",
-          "e0d7063a-9421-4148-b548-446e9fbc8566",
-          "9389d944-fdcf-4394-9ca3-4604c0af4fac"
-        ]
-      }
-    ]
-  },
-  {
-    "id": "76b3f758-6aa3-430b-bcf1-8867897692b2",
```
Comment (mmiermans) [inline app/json/slate_lineup_configs.json:583]: [Github search shows no client usage](https://github.com/search?q=org%3APocket+8867897692b2&type=code)

File: app/json/slate_lineup_configs.json [hunk 7]
```diff
@@ -550,69 +514,15 @@
...
-          "2f2a3568-901f-4655-8735-daf67e8ecc5d",
-          "505c0126-d54d-42ef-8fe7-0f6a6f24e3a5",
-          "7cb4f497-fd05-42c5-9f78-3650e9ddba21",
-          "e9df8a81-19af-48e2-a90f-05e9a37491ca"
-        ]
-      }
-    ]
-  },
-  {
-    "id": "249850f0-61c0-46f9-a16a-f0553c222800",
```
Comment (mmiermans) [inline app/json/slate_lineup_configs.json:602]: [Github search shows no client usage](https://github.com/search?q=org%3APocket+f0553c222800&type=code)

File: app/json/slate_lineup_configs.json [hunk 8]
```diff
@@ -591,28 +524,9 @@
...
-        "slates": [
-          "0737b00e-a21e-4875-a4c7-3e14926d4acf",
-          "2e3ddc90-8def-46d7-b85f-da7525c66fb1",
-          "0f322865-64e6-472d-8147-b3d6637a7d67",
-          "2f2a3568-901f-4655-8735-daf67e8ecc5d",
-          "505c0126-d54d-42ef-8fe7-0f6a6f24e3a5",
+          "b64c873e-7f05-496e-8be4-bfae929c8a04",
           "7cb4f497-fd05-42c5-9f78-3650e9ddba21",
-          "e9df8a81-19af-48e2-a90f-05e9a37491ca"
+          "6d1273a5-055e-4de0-8a5b-5f2b79d37e5c"
```
Comment (mmiermans) [inline app/json/slate_lineup_configs.json:529]: Changed slates to science, food, self-improvement based on [popularity of topic pages](https://app.mode.com/getpocket/reports/8db2aee53ddf).

File: app/json/slate_lineup_configs.json [hunk 9]
```diff
@@ -68,29 +50,13 @@
...
-          "00f0c593-3c99-4d5a-9297-c66f8b00be01",
-          "856eb5f8-da7f-43ae-ac5e-25da402af0ae",
-          "cceefa28-906e-47c5-b704-8c5b824ecd1b",
-          "651d27ab-a898-4173-9700-dd1726fbaaa4",
-          "951ae6c3-4438-458b-9936-7f3b8ff0f178",
-          "47023d21-0aa6-4f98-9077-eac21fad44f1",
-          "1a8de2be-db03-4a2f-901a-7c4c1cbd156a",
-          "f4b58337-4ab6-4563-9503-c384e773946f",
-          "f8c945b0-52fa-4203-bb31-6747245fe021",
           "b548b456-70c4-474d-a723-80d040d00fec",
```
Comment (mmiermans) [inline app/json/slate_lineup_configs.json:59]: Kept science slate based on [popularity of topic pages](https://app.mode.com/getpocket/reports/8db2aee53ddf).

File: app/json/slate_lineup_configs.json [hunk 10]
```diff
@@ -35,29 +33,13 @@
...
-          "e6c8db8d-cae7-4947-a57e-872dec639472",
-          "8aaac382-9c27-42ab-9470-ac1dfc666262",
-          "ce96db55-f32e-48f1-929f-216eb2e8c082",
-          "884414d5-502d-4e1a-8e70-1dc8eb4a345a",
-          "44dff273-3604-40e9-a0b5-bf3852581990",
-          "494eaa68-f048-4f52-bb9a-5e7d2526f5cb",
-          "e6600ffe-b65a-42d3-b570-f460f3d7326e",
-          "72ce0827-9dbc-470f-9722-39476daaeb0f",
-          "9901851d-ea76-4667-8dc7-0c1677ecb910",
           "ab6048ca-5c61-4e87-aefd-7a612b60ab7b",
```
Comment (mmiermans) [inline app/json/slate_lineup_configs.json:42]: Kept science slate based on [popularity of topic pages](https://app.mode.com/getpocket/reports/8db2aee53ddf).

File: app/json/slate_configs.json (hunk 1 above)
Comment (lfishhead) [reply app/json/slate_configs.json:264]: also unable to find engagements for this slate on snowflake

File: app/json/slate_configs.json (hunk 2 above)
Comment (lfishhead) [reply app/json/slate_configs.json:283]: found 108 users who have accessed this slate in the last 90 days.  may also have older android clients:
```
select 
    e.hashed_user_id,
    e.slate_lineup_id,
    e.os,
    count(*) as eng
from engagements as e 
where e.slate_id in ('631d8077-1462-4397-ad0a-aa340c27570a', 'ad73c061-5ac5-458f-8691-ab071f7eadd7')
and e.happened_at > current_date - 90
and slate_lineup_id is not null
group by 1, 2, 3
```

File: app/json/slate_lineup_configs.json (hunk 10 above)
Comment (lfishhead) [reply app/json/slate_lineup_configs.json:42]: there is a fallback lineup available as well.  I don't think this has large impact either way

File: app/json/slate_lineup_configs.json (hunk 6 above)
Comment (lfishhead) [reply app/json/slate_lineup_configs.json:583]: i found no web usage

**Review by lfishhead — APPROVED** [review]
i focused mainly on the json, but it all looked fine to me.  whenever we remove Ian's code I feel a little sad.  and assume our test coverage takes a hit.

**Review by jeshuaborges — APPROVED** [review]
Not technical review but @mmiermans and I synced and did a one time ranking of most engaged pocket topics and that is the new order for topics.

File: app/json/slate_configs.json (hunk 2 above)
Comment (mmiermans) [reply app/json/slate_configs.json:283]: Thanks for remembering to check Snowflake! When I change the condition to `slate_lineup_id is null` I do not get any rows back, which means that clients only request these slates as part of a lineup. I double-checked that they've both been removed from slate_lineup_configs.json.

**Pull request merged**